test/integration/ImportExportTest.php

```diff
@@ -79,30 +76,7 @@ public function setUp()
      */
     protected function getSamplePath($relPath)
     {
-        return dirname(__FILE__).DIRECTORY_SEPARATOR.str_replace('/',DIRECTORY_SEPARATOR, $relPath);
-    }
-
-    /**
-     * @expectedException oat\taoQtiItem\model\portableElement\exception\PortableElementNotFoundException
-     */
-    public function testInexistingPciLikert(){
```

Comment:
Why these method has bee removed?

Reply:
I removed them primarily because they actually test the registry implementation and not the Import / Export process.